apis/Google.Cloud.Dialogflow.V2/Google.Cloud.Dialogflow.V2/EntityType.cs

```diff
@@ -288,6 +332,7 @@ public sealed partial class EntityType : pb::IMessage<EntityType> {
       if (Kind != 0) hash ^= Kind.GetHashCode();
       if (AutoExpansionMode != 0) hash ^= AutoExpansionMode.GetHashCode();
       hash ^= entities_.GetHashCode();
+      if (EnableFuzzyExtraction != false) hash ^= EnableFuzzyExtraction.GetHashCode();
```

Comment:
Defining hashcode based on a mutable property seems wrong. Although this was happening before the introduction of `EnableFuzzyExtraction` because at least some of the other properties are mutable as well.

Reply:
Nope, that's fine - it's the same for all protos. It's fine for the hash code to change as these properties are also involved in equality checks. Yes, it means you shouldn't mutate it once you've added it to a dictionary, but it's better than not defining equality at all for protos.